coordinator/src/main/scala/filodb.coordinator/TenantIngestionMetering.scala

```diff
@@ -72,8 +72,8 @@ case class TenantIngestionMetering(settings: FilodbSettings,
             // publish a cardinality metric for each namespace
             val data = RowData.fromRowReader(rr)
             val prefix = data.group.toString.split(PREFIX_DELIM)
-            val tags = Map("metric_ws" -> prefix(0),
-                           "metric_ns" -> prefix(1),
+            val tags = Map("_tenant_ws_" -> prefix(0),
+                           "tenant_ns" -> prefix(1),
```

Comment:
1. Let's put the underscores for ns also for consistency.
2. Need the `audience="tenants"` tag as well.

Reply:
Done!